baxter_collaboration_lib/test/test_robot_arm.cpp

```diff
@@ -0,0 +1,93 @@
+#include <gtest/gtest.h>
+#include "robot_interface/arm_ctrl.h"
+
+using namespace std;
+
+
+    //Potentially expand on the testcases; check to see that all edge values have been covered
+
+TEST(ArmControlTest, testConstructorDefaultValues)
+{
+    std::string _name = "robot"; 
+    std::string _limb = "left";
+    bool _use_forces = true; 
+    //Since we are actually not using the robot, _use_robot needs to be set to 'false' even though by default it should be true
+    //Setting this to true yields a KDL chain error
+    //Perhaps this could be tested using a simulation?
+    bool _use_robot = false; 
+    bool _use_trac_ik = true;
+    bool _use_cart_ctrl = false;
+
+    ArmCtrl ac (_name, _limb, _use_robot, _use_forces, _use_trac_ik, _use_cart_ctrl);
+
+    EXPECT_EQ(ac.isCtrlRunning(), false);
+    EXPECT_EQ(START, int(ac.getState())); 
+    EXPECT_EQ("robot", ac.getName());
+    EXPECT_EQ("left", ac.getLimb());
+    EXPECT_FALSE(ac.isRobotUsed());
+    EXPECT_TRUE(ac.useForces()); 
+    EXPECT_TRUE(ac.useTracIK());
+    EXPECT_FALSE(ac.useCartCtrl()); 
+}
+
+TEST(ArmControlTest, testConstructorCustomValues) 
+{
+    std::string _name = "robot"; 
+    std::string _limb = "left";
+    //overrides default constructor values for these variables; these values are the opposite of the default values bool _use_forces = false; 
```

Comment:
split this in two lines. In general, try to avoid having long lines. The guideline says to use a max of 80 columns, but it is not mandatory to be always below 80 columns.